dialect/sql/schema/migrate.go

```diff
@@ -235,6 +236,14 @@ type changes struct {
 // changeSet returns a changes object to be applied on existing table.
 // It fails if one of the changes is invalid.
 func (m *Migrate) changeSet(curr, new *Table) (*changes, error) {
+	if curr == nil {
+		return nil, errors.New("current state could not be determined during change set generation")
+	}
+
+	if new == nil {
+		return nil, errors.New("determined state could not be determined during change set generation")
+	}
```

Comment:
Use this format: `sql/schema: determined state could ...`.
 
Also, maybe add the table name as well?

Reply:
ok. Would you like wrappers for these package name identifiers usable for `github.com/pkg/errors.Wrap`? Could do this in a later PR.